airbyte-cdk/java/airbyte-cdk/db-sources/src/testFixtures/java/io/airbyte/cdk/testutils/TestDatabase.java

```diff
@@ -95,7 +95,7 @@ public T with(String fmtSql, Object... fmtArgs) {
    * object. This typically entails at least a CREATE DATABASE and a CREATE USER. Also Initializes the
    * {@link DataSource} and {@link DSLContext} owned by this object.
    */
-  final public T initialized() {
+  public T initialized() {
```

Comment:
Looks like this is only used to start the container in the call to XXTestDatabase.initialized. Why can't that be done in the constructor of those XXXTestDatabase classes?